src/js/letters/utils/helpers.js

```diff
@@ -37,6 +37,11 @@ export function apiRequest(url, optionalSettings = {}, success, error) {
     .then(success, error);
 }
 
+export const characterOfServiceContent = {
```

Comment:
No, this was just a placeholder. I just looked up the types in the EVSS swagger docs and updated https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3689.

Possible values are ['HONORABLE', 'OTHER_THAN_HONORABLE', 'UNDER_HONORABLE_CONDITIONS', 'GENERAL', 'UNCHARACTERIZED', 'UNCHARACTERIZED_ENTRY_LEVEL', 'DISHONORABLE']

I can update these and guess at the content (for the most part just follow the pattern e.g., `under_honorable_conditions` -> "Under honorable conditions" and verify with product/UX later.